libsolidity/interface/CompilerStack.cpp

```diff
@@ -1024,6 +1025,36 @@ Json::Value CompilerStack::methodIdentifiers(string const& _contractName) const
 	return methodIdentifiers;
 }
 
+Json::Value CompilerStack::errorIdentifiers(string const& _contractName) const
+{
+	if (m_stackState < AnalysisPerformed)
+		solThrow(CompilerError, "Analysis was not successful.");
+
+	Json::Value errorIdentifiers(Json::objectValue);
+	for (auto const& it: contractDefinition(_contractName).interfaceErrors())
```

Comment:
This is not an iterator.
```suggestion
	for (ErrorDefinition const* error: contractDefinition(_contractName).interfaceErrors())
```